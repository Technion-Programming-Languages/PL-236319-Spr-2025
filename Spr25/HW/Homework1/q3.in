LOAD
4
WRITE_STACK
DUP
LOAD
1
SUB
DUP
JNZ
1
