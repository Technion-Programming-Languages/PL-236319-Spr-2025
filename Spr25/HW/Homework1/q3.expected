Stack: 4 
Stack: 4 3 
Stack: 4 3 2 
Stack: 4 3 2 1 
